### ISSUE TITLE
Wishlist. Add the mapping runtime tests combined in @Stories

### ISSUE THREAD

**Issue by choojoykin** [allure-framework/allure1#899]
The report (ver. 1.5.0 as last) displays the total run time of all tests that are running, for example, in parallel execution from maven goals execute command line. As a result, it is difficult to see how long it took the implementation of a @ Stories as a test group.

**Comment by baev** [comment]
Already done in Allure2. See https://github.com/allure-framework/allure2/releases/tag/2.0-BETA4
